attendance_app/models.py

```diff
@@ -1,6 +1,56 @@
 from django.db import models
 
 # Create your models here.
+
+#Manager Class for UserProfile
+
+class QuestionManager(models.Manager):
+    def createQuestion(self,questionText):
+        self.create(questionText=questionText)
+
+class Question(models.Model):
+    questionText = models.CharField(max_length = 300)
+    objects = QuestionManager()
+
+class AnswerManager(models.Manager):
+    def createAnswer(self, question, answerText):
+        self.create(question=question,answerText=answerText)
+         
+
+class Answer(models.Model):
+      question = models.ForeignKey(Question, on_delete = models.SET_NULL, null = True)
+      answerText = models.CharField(max_length = 300)     
+      objects = AnswerManager()                  
+
+class UserProfileManager(models.Manager):
+    #create new userprofile if not existed 
+    def createUserProfile(self, username, chat_url, first_name, last_name, email, created_on, role):
```

Comment:
created_on should be generated by the model
if the user profile already exists, we should update: first_name, last_name, email and role
For long parameter list for constructor like this, try to use Builder pattern:
 Foo foo = new Foo()
          .configBar(anything)
          .configBaz(something, somethingElse)
          // and so on